docs/core/windows-prerequisites.md

```diff
@@ -46,18 +46,51 @@ For download links and more information, see [.NET downloads](https://dotnet.mic
 >
 > In addition to KB2999226, make sure you also have [KB2533623](https://support.microsoft.com/help/2533623/microsoft-security-advisory-insecure-library-loading-could-allow-remot) installed. If you don't have this update installed, you'll see an error similar to the following when you launch a .NET Core application: `The library hostfxr.dll was found, but loading it from C:\<path_to_app>\hostfxr.dll failed`.
 
-## Prerequisites for .NET Core 3.0 Preview 3
+## Prerequisites with Visual Studio
+    
+You can use any editor to develop .NET Core applications using the .NET Core SDK. Visual Studio provides an integrated development environment for .NET Core apps on Windows.
```

Comment:
Should we qualify Visual Studio as "Visual Studio 2017 or later". That's protection against someone assuming an older version like VS 2015 could be used for Core development.